docker/Dockerfile

```diff
@@ -7,16 +7,6 @@ FROM $PARENT_IMAGE AS garage-18.04
 # > At the moment, setting "LANG=C" on a Linux system *fundamentally breaks Python 3*, and that's not OK.
 ENV LANG C.UTF-8
```

Comment:
BTW, I don't think the C.UTF-8 logic is necessary anymore.

Reply:
Yeah -- i think this was fixed in Py36

Reply:
Looks like it was fixed in 3.7 https://bugs.python.org/issue19846#msg308567

Reply:
Ah, i guess we'll just have to live with it then.